schemas/answers/dropdown.json

```diff
@@ -0,0 +1,58 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema",
+  "answer": {
+    "type": "object",
+    "properties": {
+      "id": {
+        "$ref": "../common_definitions.json#/id"
+      },
+      "q_code": {
+        "$ref": "../common_definitions.json#/q_code"
+      },
+      "guidance": {
+        "$ref": "../common_definitions.json#/guidance"
+      },
+      "type": {
+        "type": "string",
+        "enum": [
+          "Dropdown"
+        ]
+      },
+      "mandatory": {
+        "type": "boolean"
+      },
+      "label": {
+        "type": "string"
+      },
+      "description": {
+        "type": "string"
+      },
+      "preselected_option": {
```

Comment:
I don't really like how this is used to inject a value into the options list.
We shouldn't ever want to pre-select a value for someone

@ajmaddaford what are you views on this?

Reply:
The SDC spec says that the initially-selected option should always be the null "Select an option".

I guess this might be necessary later for pre-population though?

Reply:
I think the AC on the trello card wants this value to be configurable:
https://trello.com/c/lZqYhhjw/1328-add-dropdown-input-type-m

Reply:
@nealedj if its not needed now then we shouldn't have it in the code.
@shahi645 Maybe it could be the Label?

Reply:
@jonnyshaw89 Ah I misunderstood what this actually does. So this is just where the default option text gets set? Surely something like `default_text` makes it pretty clear?

Label could be confused with the form input label.

Reply:
@MebinAbraham has asked Charlotte whether it is required.

Reply:
I've followed this up with Charlotte, and she said that she will update the ticket as we do not need to override the text. It should just always just say "Select an answer".
Ill sort this shortly.